android/build.gradle

```diff
@@ -69,7 +69,7 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.okta.android:oidc-androidx:1.0.17'
+    implementation 'com.okta.android:oidc-androidx:1.0.19'
```

Comment:
This is the incorrect dep. https://github.com/okta/okta-oidc-android/releases/tag/1.0.19 @NikitaAvraimov-okta

Reply:
Correct dependency is `implementation 'com.okta.android:okta-oidc-android:1.0.19'`